testsuite/MDAnalysisTests/analysis/test_dihedrals.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
```

Comment:
The copyright header should also be added here

Reply:
pylint will also issue a warning here to add `from __future__ import absolute_import` as the top most import